pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -219,24 +219,33 @@ type PermissionClaim struct {
 	IdentityHash string `json:"identityHash,omitempty"`
 }
 
+const (
+	ResourceSelectorAllNamespaces = "*"
+)
+
+// ResourceSelector identifies the objects to be included within a PermissionClaim either by name or namespace.
+//
 // +kubebuilder:validation:XValidation:rule="has(self.__namespace__) || has(self.name)",message="at least one field must be set"
```

Comment:
this validation causes kcp to start:
```
E0120 09:53:55.327990   45895 bootstrap.go:63] "failed to bootstrap system CRDs, retrying" err=<
	could not bootstrap CRDs: [error creating CRD apiexports.apis.kcp.io: CustomResourceDefinition.apiextensions.k8s.io "apiexports.apis.kcp.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[permissionClaims].items.properties[resourceSelector].items.x-kubernetes-validations[0].rule: Invalid value: apiextensions.ValidationRule{Rule:"has(self.__namespace__) || has(self.name)", Message:"at least one field must be set"}: compilation failed: ERROR: <input>:1:4: undefined field '__namespace__'
	 | has(self.__namespace__) || has(self.name)
	 | ...^, error creating CRD apibindings.apis.kcp.io: CustomResourceDefinition.apiextensions.k8s.io "apibindings.apis.kcp.io" is invalid: [spec.validation.openAPIV3Schema.properties[spec].properties[permissionClaims].items.properties[resourceSelector].items.x-kubernetes-validations[0].rule: Invalid value: apiextensions.ValidationRule{Rule:"has(self.__namespace__) || has(self.name)", Message:"at least one field must be set"}: compilation failed: ERROR: <input>:1:4: undefined field '__namespace__'
	 | has(self.__namespace__) || has(self.name)
	 | ...^, spec.validation.openAPIV3Schema.properties[status].properties[exportPermissionClaims].items.properties[resourceSelector].items.x-kubernetes-validations[0].rule: Invalid value: apiextensions.ValidationRule{Rule:"has(self.__namespace__) || has(self.name)", Message:"at least one field must be set"}: compilation failed: ERROR: <input>:1:4: undefined field '__namespace__'
	 | has(self.__namespace__) || has(self.name)
	 | ...^, spec.validation.openAPIV3Schema.properties[status].properties[appliedPermissionClaims].items.properties[resourceSelector].items.x-kubernetes-validations[0].rule: Invalid value: apiextensions.ValidationRule{Rule:"has(self.__namespace__) || has(self.name)", Message:"at least one field must be set"}: compilation failed: ERROR: <input>:1:4: undefined field '__namespace__'
	 | has(self.__namespace__) || has(self.name)
	 | ...^]]
 >
```

Reply:
fixed by setting `has(self.namespaces)` however we have to modify the unit tests.